classes/Kohana/Jam/Price.php

```diff
@@ -250,19 +252,19 @@ public function is($operator, $value)
 		switch ($operator) 
 		{
 			case Jam_Price::EQUAL_TO:
-				$result = ($this->amount() == $value);
+				$result = (abs($this->amount() - $value) < Jam_Price::EPSILON);
 			break;
 			case Jam_Price::GREATER_THAN:
-				$result = ($this->amount() > $value);
+				$result = (($this->amount() - $value) >= Jam_Price::EPSILON);
 			break;
 			case Jam_Price::LESS_THAN:
-				$result = ($this->amount() < $value);
+				$result = (($value - $this->amount()) >= Jam_Price::EPSILON);
 			break;
 			case Jam_Price::GREATER_THAN_OR_EQUAL_TO:
-				$result = ($this->amount() >= $value);
+				$result = (($value - $this->amount()) < Jam_Price::EPSILON);
 			break;
 			case Jam_Price::LESS_THAN_OR_EQUAL_TO:
-				$result = ($this->amount() <= $value);
+				$result = (($this->amount() - $value) < Jam_Price::EPSILON);
```

Comment:
@hkdobrev Well it all depends on if we want to include the interval border in the equal comparison. Since not including it gives us a bit more precision i've chosen to go this way. Basically if the difference is exactly EPSILON the two prices are not equal.